### PR TITLE
Added card for "Doctors as volunteers"

### PR DESCRIPTION
**Pull request by ozer619**
Added card for "Doctors as volunteers" in give-ur-time section

Screenshot_20210827_223459

### CODE REVIEW

File: components/DoctorsAsVoulenters.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Card, { CardParagraph, CardTitle } from './Card';
+
+function DoctorsAsVolunteer() {
+  return (
+    <Card backgroundImageSource="/images/Volunteer.jpg">
+      <CardTitle>{'Doctors as Volunteer'}</CardTitle>
```
Comment (audreyfeldroy) [inline components/DoctorsAsVoulenters.js]:
```suggestion
      <CardTitle>{'Doctors as Volunteers'}</CardTitle>
```

File: components/DoctorsAsVoulenters.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        }
+      </CardParagraph>
+      <CardParagraph>
+        {'Scheduling is flexible and based on your availability.'}
+      </CardParagraph>
+
+      <CardParagraph>{'Requirements:'}</CardParagraph>
+
+      <ul className="list-disc text-left pl-6">
+        <li>{'Current Medical Board of California license'}</li>
```
Comment (audreyfeldroy) [inline components/DoctorsAsVoulenters.js]:
```suggestion
        <li>{'Current medical board license for your region and country'}</li>
```

Comment (audreyfeldroy) [reply components/DoctorsAsVoulenters.js]: Per discussion today with Alondra, we decided to expand this globally

File: pages/give-your-time.js [hunk 3]
```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 
+import DoctorsAsvolunteer from '../components/DoctorsAsVoulenters';
```
Comment (audreyfeldroy) [inline pages/give-your-time.js:3]:
```suggestion
import DoctorsAsVolunteers from '../components/DoctorsAsVolunteers';
```

**Review by audreyfeldroy — APPROVED** [review]
Hi @ozer619, really great work. @RedFox0x20 thanks for the review and for helping @ozer619 so kindly. I've made some text edits and am approving this for merge.

**Comment by audreyfeldroy** [comment]
Hmm, I broke something. Looking into it.

**Comment by audreyfeldroy** [comment]
Welcome to the project and congrats on your first merged PR @ozer619!

**Comment by ozer619** [comment]
> Welcome to the project and congrats on your first merged PR @ozer619!

Thanks for guiding me, I'm looking forward to contribute more on this project.

**Pull request merged**